### DISCUSSION TITLE
Silent Installation options for /tasks

### DISCUSSION THREAD

**Discussion by scottchamings** [git-for-windows/git#2999] [Q&A]
Looking at automating installation for git for windows from the command line and want to set the options displayed during the installation.

I have tried the /saveinf: option without success (no file got generated)

I also believe you can use the /mergetasks option to setup options as well but I cant seem to find any information on the setup tasks that can be specified for git for windows

I have tried using innounp050 to reverse engineer the installer and extract out the ISS fie but also couldnt find anything there.

Does anyone have any more ideas on where I can look to setup a silent custom installation file for git for windows?

I know about  /VERYSILENT but interested in setting the options selected during the installation.

**Comment by scottchamings** [comment]
Update - will be checking out https://github.com/git-for-windows/build-extra/tree/main/installer for ISS options and tasks

**Comment by dscho** [comment]
There is no need to reverse-engineer, the ISS file is here: https://github.com/git-for-windows/build-extra/blob/main/installer/install.iss
